app/controllers/subscriptions_controller.rb

```diff
@@ -62,6 +62,11 @@ def update
     render json: { subscription: subscription }, status: :ok
   end
 
+  def get_latest_matching
```

Comment:
I think it'd be a nice thing if this matches the url route and _probably_ oughtn't include 'get' as the method doesn't need to include the HTTP verb (I can't find any style guidelines on controller method names so these are just my opinions). I'd go with making them all `latest_matching` as that's clearest 😃